list

```diff
@@ -332,6 +340,7 @@ deb.li
 deli.bz
 dell.to
 deloi.tt
+demo.condize.net
```

Comment:
This one doesn't look like one, and I didn't see any working examples there.

Reply:
Original link (https://demo.condize.net/shortny/x7WrP) still works for me, takes to a 10 second redirect page, then 304 redirects.
Archive: https://archive.is/0P5zh